src/icons/fileIcons.ts

```diff
@@ -1473,5 +1476,9 @@ export const fileIcons: FileIcons = {
       fileNames: ['jsconfig.json'],
       fileExtensions: ['jsconfig.json'],
     },
+    {
+      name: 'maven',
+      fileNames: ['extensions.xml', 'maven.config', 'jvm.config'],
```

Comment:
As I'm not that knowledged with Maven, I'd like to know if the "jvm.config" file is really maven specific? Same question for "extensions.xml". Even if "extensions.xml" has something to do with Maven in a Maven-Project, I think it would be better to keep the XML icon here. There might be other projects outside where this filename could be used without Maven context, and then it would be confusing if this file will show a Maven icon.

Reply:
`jvm.config` is always associated with Apache Maven instead of Java/JVM. As for `extensions.xml` you are unfortunately right - it is used for multiple purposes as a generic config file.

Again a situation where https://github.com/microsoft/vscode/issues/12493#issuecomment-864600866 would come in useful..

Reply:
Resolved in e4c45b2 @PKief